fixtures/lint/casing-fail.yaml

```diff
@@ -21,11 +21,11 @@ paths:
       tags:
         - Test
       parameters:
-        - name: some_query_param
+        - name: someQueryParam
```

Comment:
Why is this "going the other direction" from snake to camel?

Reply:
Same comment as below - it's a failure test case!